app/models/equipment.rb

```diff
@@ -0,0 +1,9 @@
+class Equipment < ApplicationRecord
+  has_many :equipment_materials
```

Comment:
```suggestion
  has_many :equipment_materials, dependent: :destroy
```
porque si no creo que puede fallar cuando elimines equipment